modules/src/player.js

```diff
@@ -19,799 +19,5 @@
-              if (Object.keys(standaloneElems).length > 0 &&
-                standaloneElems.hasOwnProperty(keyValue) &&
-                Object.keys(standaloneElems[keyValue]).length > 0
-              ) {
-                $.each(Object.keys(standaloneElems[keyValue]),
-                  function(keyIndex, elemKey) {
-                    (standaloneElems[keyValue][elemKey].hbs) && $content.append(
-                      renderElement(
-                        standaloneElems[keyValue][elemKey].hbs,
-                        standaloneElems[keyValue][elemKey].templateData,
-                        true,
-                      ),
-                    );
-                  });
-              }
-            }
-          });
-      }
-    }
-  }
-
-  PlayerHelper
-    .init(widgetData, elements)
-    .then(({widgets, countWidgetElements}) => {
-      if (Object.keys(widgets).length > 0) {
-        let _countWidgetElements = 0;
-        Object.keys(widgets).forEach(function(widgetKey) {
-          const widget = widgets[widgetKey];
-
-          // Check if we have template from templateId or module
-          // and set it as the template
-          let $template = null;
-          let moduleTemplate = false;
-          if ($('#hbs-' + widget.templateId).length > 0) {
-            $template = $('#hbs-' + widget.templateId);
-          } else if ($('#hbs-module').length > 0) {
-            $template = $('#hbs-module');
-            moduleTemplate = true;
-          }
-
-          // Save widgetData to xic
-          xiboIC.set(widget.widgetId, 'widgetData', widget);
-
-          let hbs = null;
-          // Compile the template if it exists
-          if ($template && $template.length > 0) {
-            hbs = Handlebars.compile($template.html());
-          }
-
-          const $content = $('#content');
-
-          if ((widget.templateId === 'elements' ||
-              (widget.templateId === null && (
-                Object.keys(widget.elements.groups).length > 0 ||
-                Object.keys(widget.elements.standalone).length > 0
-              ))) && typeof widget.elements !== 'undefined') {
-            widget.items = [];
-
-            _countWidgetElements++;
-
-            initPlayerElements(widget);
-
-            if (countWidgetElements === _countWidgetElements) {
-              // Run xiboLayoutScaler to scale the content
-              $content.xiboLayoutScaler(Object.assign(
-                widget.properties,
-                globalOptions,
-                {duration: widget.duration},
-              ));
-            }
-          } else {
-            initStaticTemplates(
-              $template,
-              $content,
-              moduleTemplate,
-              hbs,
-              widget,
-              widget.data,
-              widget.showError,
-              widget.errorMessage,
-              widget.data,
-            );
-          }
-        });
-      }
-    });
+  XiboPlayer.init(widgetData, elements);
```

Comment:
Do we still need this file? Or could we just use `xibo-player.js` to initialize instead?

Reply:
I've thought about this as well 😅 Should we just add it after the `module.exports` on `xibo-player.js`?